modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/index.js

```diff
@@ -179,7 +179,8 @@ export function loadData(
 	const formattedUrl = apiURL.replace(
```

Comment:
I wonder if this would be more readable if instead of using `replace()` we manipulate the url after we construct it with `URL()`. If I understand this right, what you want with this replace is

1. Grab the filter value
2. Remove the filter parameter

Maybe we could do

```js
const url = new URL(apiURL, themeDisplay.getPortalURL());

const providedFilters = url.searchParams.get('filter'); // not sure we need decodeURIComponent with URLSearchParams.get()

url.searchParams.delete('filter');

...
```

Reply:
Great! I'll check it out right now :)